contracts/PetPark.sol

```diff
@@ -3,5 +3,58 @@ pragma solidity ^0.8.0;
 
 
 contract PetPark {
+    enum AnimalType{None,Fish,Cat,Dog,Rabbit,Parrot}
```

Comment:
It is better to use a formatting plugin in vscode. You can try using this plugin for vscode - https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity

Reply:
Thanks just added it works wonder earlier I used solidity by juan blanco but this one is much better